src/recon_buildblock/Reconstruction.cxx

```diff
@@ -190,14 +190,6 @@ set_disable_output(bool _val)
     this->_disable_output = _val;
 }
 
-template <typename TargetT>
-void
-Reconstruction<TargetT>::
-set_enable_output(bool _val)
-{
-    this->_disable_output = _val;
-}
-
```

Comment:
I checked and this function is not used anywhere within the repo. Also, it is actually wrong: says it sets enable_output, but actually sets disable_output. So for what this function does, there is already a 1:1 copy in "set_disable_output" (line 188), which is also correctly named! Therefore, I thought of this as a bugfix, rather than breaking backwards compatibility :-)

But happy to do the suggested changes if you think it's necessary.

Reply:
ok. let's cut it then!